### PR TITLE
Halt connection if no auth header provided

### PR DESCRIPTION
**Pull request by ambuznego**
```
➜  console git:(appsignalError) ✗ mix test
Compiling 58 files (.ex)
warning: function Console.AccessTokenDecoder.MockDecodeAccessToken.decode_conn_access_token/1 is undefined (module Console.AccessTokenDecoder.MockDecodeAccessToken is not available)
Found at 2 locations:
  lib/console_web/channels/user_socket.ex:24
  lib/console_web/plug/verify_access_token.ex:67

...................................................

Finished in 1.6 seconds
51 tests, 0 failures
```

### CODE REVIEW

File: lib/console_web/plug/verify_access_token.ex [hunk 1]
```diff
@@ -47,28 +47,40 @@ defmodule ConsoleWeb.Plug.VerifyAccessToken do
...
+            })
+          )
+          |> halt()
+      _ ->
+        token = conn
+        |> get_req_header("authorization")
+        |> List.first()
+        |> String.replace("Bearer ", "")
 
+      case @access_token_decoder.decode_conn_access_token(token) do
```
Comment (vicmgs) [inline lib/console_web/plug/verify_access_token.ex:67]: we aren't missing an end right?

**Review by vicmgs — APPROVED** [review]

File: lib/console_web/plug/verify_access_token.ex (hunk 1 above)
Comment (ambuznego) [reply lib/console_web/plug/verify_access_token.ex:67]: no sir 
image

Comment (vicmgs) [reply lib/console_web/plug/verify_access_token.ex:67]: LOL thanks for the clear highlighting!

Comment (ambuznego) [reply lib/console_web/plug/verify_access_token.ex:67]: lol my indent is off so thanks for catching that

**Pull request merged**